LoopKitUI/Views/ListMultipleSelection.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  ListMultipleSelection.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-02-27.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+public struct ListMultipleSelection<Item>: View where Item: Hashable, Item: CustomStringConvertible {
```

Comment:
Constraining to `CustomStringConvertible` is unnecessary; `String.init(describing:)` works on all types.

Reply:
Yes, but I'm intending to force the user of this View to ensure they have the correct string to be displayed with describing, instead of whatever will be displayed by default. If this not the right way to go about this?